diagnostic.md

```diff
@@ -7,40 +7,51 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+to work on this repo locally, we clone it onto our desired directory by using the
+command: git clone git@github.com:TronB/git-diagnostic.git
+
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+type in the command: git status, to see if there are any files that have been
+modified
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+You can type in git checkout -b response to both create the response branch as
+well as switching over to that specific branch. To JUST create a branch, you
+would input the command: git branch *NAME YOUR BRANCH HERE*. Additionally,
+if you want to ONLY switch branches you would type: git checkout *ENTER BRANCH HERE*
+
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git status
 ```
 
 5.We want to prepare a change for a commit by adding a file to the staging
     area. What command do we use to stage a file named `diagnostic.md`?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git add diagnostic.md
 ```
 
 6.Once `diagnostic.md` is staged, we have to make a commit by `git commit`.
 What are the two formatting items you **need** to make up your commit message?
 
-<!-- Remove this comment and place your answer here. -->
+There should be a commit message that says you can use the command:
```

Comment:
A commit message is what you write when making a commit, not what the console tells you after making a commit. In this case, you would something like a headline and a description.